core/src/apps/misc/cipher_key_value.py

```diff
@@ -39,7 +39,7 @@ async def cipher_key_value(msg: CipherKeyValue) -> CipheredKeyValue:
         await confirm_action(
             "cipher_key_value",
             title,
-            description=msg.key,
+            description=TR.misc__enable_labeling,
```

Comment:
This will not work as intended when we are not using cipher_key_value to enable labeling

Reply:
fixed in force-push